content/testsuite/testsuite.go

```diff
@@ -280,6 +282,39 @@ func checkResumeWriter(ctx context.Context, t *testing.T, cs content.Store) {
 	}
 }
 
+func checkCommitExists(ctx context.Context, t *testing.T, cs content.Store) {
+	c1, d1 := createContent(256)
+	if err := content.WriteBlob(ctx, cs, "c1", bytes.NewReader(c1), ocispec.Descriptor{Digest: d1}); err != nil {
```

Comment:
Looks like this line will also need to have an import for `ocispec`

Reply:
Yeah; been sitting on it as I wanted to make sure if it was just the import as it seems the write style has changed since 1.1.x branch, and maybe it's easier to make the test write in the old style (without the ocispec import?)